### PR TITLE
Update configuration related docs

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Update configuration docs to reflect changes introduce in https://github.com/kubeshop/botkube/pull/648

**Preview URL:** https://multi-config.botkube-docs.pages.dev/

Helm chart params will be updated once https://github.com/kubeshop/botkube/pull/648 and https://github.com/kubeshop/botkube-docs/pull/99 will be merged.

## Related issue(s)

- Fixes https://github.com/kubeshop/botkube/issues/643

### CODE REVIEW

**Review by pkosiec — COMMENTED** [review]
A few small comments 🙂

File: content/configuration/_index.md [hunk 1]
```diff
@@ -2,23 +2,119 @@
...
 draft: false
-weight: 10
+weight: 20
 ---
 
-BotKube backend allows you to specify the [resource](/configuration/resource) and [communication](/configuration/communication) settings. Check the individual documents for more detailed explanation on how to provide those configurations.
+BotKube backend allows you to specify [resource](/configuration/resource), [communication](/configuration/communication), and [global](/configuration/global) settings. Check the related documents for more detailed explanation.
+
+The configuration settings are read from two sources:
+- First, the configuration files specified by the `BOTKUBE_CONFIG_PATHS` environment variable or `--config/-c` flag. For example:
```
[the remark this answers is not in the document]
Comment (mszostok) [reply content/configuration/_index.md:11]: Hm.. the interesting thing is that it's only on Cloudflare 🤔 when tested locally it worked properly

File: content/configuration/helm-chart-parameters.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Helm chart parameters
-weight: 40
```
Comment (pkosiec) [inline content/configuration/helm-chart-parameters.md:3]: Please synchronize the Helm chart params as a part of this PR. Thanks!

File: content/examples/_index.md [hunk 3]
```diff
@@ -15,33 +15,33 @@ The default notification type is **short**
 
 #### Resource created
 
-![](/images/create_sh.png "Slack notiftype=short")
-![](/images/create.png "Slack notiftype=long")
+![](/images/create_sh.png "Slack notifyType=short")
```
Comment (pkosiec) [inline content/examples/_index.md:18]: While it's not really important, please also change `notifyType` to `notification.type` for all the images. Thanks!

**Review by pkosiec — APPROVED** [review]
LGTM, but please address the comments before merge. Thanks!

File: content/configuration/helm-chart-parameters.md (hunk 2 above)
Comment (mszostok) [reply content/configuration/helm-chart-parameters.md:3]: Yep, I mentioned that in the desc, as it requires other stuff to be merged before

**Pull request merged**